assets/xml/objects/object_d_hsblock.xml

```diff
@@ -1,7 +1,7 @@
 ﻿<Root>
     <File Name="object_d_hsblock" Segment="6">
```

Comment:
Looking at it in vbindiff, the objects are actually identical, so you can just bring all the names over.